resources/tickets.py

```diff
@@ -92,3 +92,19 @@ def post(self):
 
         ticket.save_to_db()
         return ticket.json(), 201
+
+    @pm_level_required
+    def delete(self):
+        self.parser.add_argument("ids", action="append")
+        data = Tickets.parser.parse_args()
+
+        if not ("ids" in data and type(data["ids"]) is list):
+            return {"message": "Ticket IDs missing in request"}, 400
```

Comment:
This is a validation, and should be handled in the schema. Since we wouldn't be using our normal model actions to make the update you would just call validate on the schema directly from here.
`TenantModel.validate(TenantSchema, request.json)`.

Reply:
"ids" is not part of the TicketModel so I am not sure why we would be validating it with the Model's schema, or am I misunderstanding something? If I put a validator on here with 
`TicketModel.validate(TicketSchema, request.json)`
I get 
```
File "C:\Users\cboe1\.virtualenvs\dwellinglybackend-vISDP1tA\Lib\site-packages\marshmallow\schema.py", line 1116, in _invoke_field_validators
    raise ValueError(
ValueError: "ids" field does not exist.```

Reply:
I don't think its necessary to validate that the id exists before deleting it. And while "ids" are not part of the ticket table. The `id` in the list of 'ids' are. Marshmallow doesn't allow validations on any fields that are not declared. Here we can tell Marshmallow that there is a field called ids and it will allow us to validate it. 

Since you mentioned that you rather just ignore it when an id does not exist, then I don't think we need the validation and can simply execute the batch delete statement.

Reply:
oh I see, we're checking to see if the request is in the right format. Yeah this should be moved to validation. I'm not too concerned about this type of validation though. I would just delete this. If we were having other parties consume the API then it might be worth validating so that we can respond with an appropriate failure message.